form-app/src/components/formBody.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react'
+import '../style/formBody.css'
+import image from '../images/shecodes.jpg'
+import axios from "axios"
+import {nanoid} from "nanoid"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faEnvelope, faMapPin, faUser } from '@fortawesome/free-solid-svg-icons'
+
+const FormBody = () => {
+    const [form, setForm] = useState({ name: '', email: '', subject: '', message: '' })
+    const handelChange = (event) => {
+        setForm({
+            ...form,
+            [event.target.id]: event.target.value,
+        });
+    };
+
+const handelSubmit = (event) => {
+    const post = {
+        id: nanoid(),
+        name: form.name,
+        email: form.email,
+        subject: form.subject,
+        message: form.message
+    };
+    axios.post(`https://my-json-server.typi.com/tundeojediran/contacts-api-server/inquiries`, post)
```

Comment:
This is not the correct API endpoint. Here is the corrrect one 
https://my-json-server.typicode.com/tundeojediran/contacts-api-server/inquiries